deploy.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+import argparse
+import os
+import shutil
+import signal
+import subprocess
+import sys
+import time
+import zipfile
+
+DEPLOY_DIR = "/home/julius/server-snapshots"
+# DEPLOY_DIR = "/home/server-snapshots"
+CURRENT_DEPLOY_DIR = os.path.join(DEPLOY_DIR, "whereuat-server")
+BACKUP_DIR = os.path.join(DEPLOY_DIR, "_whereuat-server")
+
+
+# Make a new directory in the deploying directory and return the path of the new
+# directory.
+def makeNewDir():
+  new_dir = "{:0.0f}".format(time.time())
+  new_path = os.path.join(DEPLOY_DIR, new_dir)
+  os.mkdir(new_path)
+  return new_path
+
+
+# Delete the old backup.
+def removeBackup():
+  try:
+    shutil.rmtree(BACKUP_DIR)
+  except OSError as e:
+    if e.errno == 2:
+      print "Backup directory {} doesn't exist. Creating it now.".format(
+        BACKUP_DIR)
+      os.mkdir(BACKUP_DIR)
+
+
+# Rename the current deploy to be the backup.
+def makeCurrentDeployBackup():
+  try:
+    shutil.move(CURRENT_DEPLOY_DIR, BACKUP_DIR)
+  except IOError as e:
+    if e.errno == 2:
+      print "Current deploy directory {} doesn't exist.".format(
+        CURRENT_DEPLOY_DIR)
+
+
+# Unzip _zf_ and put it in _dest_.
+def unzip(zf, dest):
+  with open(zf) as zipped, zipfile.ZipFile(zf, 'r') as zip_ref:
+    zip_ref.extractall(DEPLOY_DIR)
```

Comment:
Move this to a temp folder instead of directly to the deploy directory to avoid collisions (remember to clean up temp folder afterwards)